spec/requests/cloud_databases_spec.rb

```diff
@@ -53,5 +55,31 @@
         expect(response).to have_http_status(:ok)
       end
     end
+
+    context 'DELETE cloud database' do
+      it "deletes the cloud database" do
+        api_basic_authorize(resource_action_identifier(:cloud_databases, :delete))
+
+        delete(api_cloud_database_url(nil, cloud_database))
+        expect(response).to have_http_status(:no_content)
+      end
+    end
+
+    context 'PATCH cloud database' do
+      it "updates the cloud database name" do
+        api_basic_authorize(resource_action_identifier(:cloud_databases, :edit))
+
+        patch(api_cloud_database_url(nil, cloud_database), :params => [{:name => "test-db-updated"}])
+
+        expected = {
+          "success"   => true,
+          "message"   => "Updating Cloud Database id: #{cloud_database.id} name: '#{cloud_database.name}'",
+          "task_id"   => anything,
+          "task_href" => a_string_matching(api_tasks_url)
+        }
+        expect(response.parsed_body).to include(expected)
+        expect(response).to have_http_status(:ok)
```

Comment:
I think it should be something like this:

```suggestion
        expect_single_action_result(:success => true, :message => /Updating Cloud Database id: #{cloud_database.id}/, :task_id => true)
```